hodl/exchanges/PoloniexHelperAPI.py

```diff
@@ -47,6 +50,21 @@ def print_poloniex_ascii():
           '| | | (_) | | (_) | | | | |  __/>  < \n'
           '\_|  \___/|_|\___/|_| |_|_|\___/_/\_\\')
 
+def print_poloniex_trade_history():
+    if settings.poloniex == 'on':
```

Comment:
Considering that when we use the `-poloTradeHist` argument we are explicitly asking for Poloniex info, I don't see any benefit to having an off functionality for this. _Removing this settings check would also be more consistent with the behavior of existing code._ 

Thoughts?